Penteract/Assets/Scripts/Player.h

```diff
@@ -52,20 +54,24 @@ class Player : public Character {
 	virtual void CheckCoolDowns(bool noCooldownMode = false) {}
 	int GetMouseDirectionState();
 	bool IsActive();
-
+	void IncreaseUltimateCounter();
 public:
 	float rangedDamageTaken = 1.0f;
 	float meleeDamageTaken = 1.0f;
 	float attackSpeed = 1.0f;
 	float attackCooldownRemaining = 0.f;
-	bool shooting = false;
+	bool canShoot = false;
 	float3 lookAtMousePlanePosition = float3(0, 0, 0);
 	ComponentCamera* lookAtMouseCameraComp = nullptr;
 	CameraController* cameraController = nullptr;
 	ComponentAudioSource* playerAudios[static_cast<int>(AudioPlayer::TOTAL)] = { nullptr };
 	float3 facePointDir = float3(0, 0, 0);
 	MovementDirection movementInputDirection = MovementDirection::NONE;
 	ComponentTransform* playerMainTransform = nullptr;
+	int ultimateChargePoints = 0;
+	int ultimateChargePointsTotal = 10;
```

Comment:
If this value is not going to change in the whole gameplay, set it as constexpr

Reply:
using const, contexpr generates an error here.